templates/sios-datakeeper-master.template

```diff
@@ -898,15 +813,7 @@
                         "Ref": "QSS3BucketName"
                     },
                     "QSS3KeyPrefix": {
-                        "Fn::Join": [
-                          	"/",
-                           [
-                             	{
-                                	"Ref": "QSS3KeyPrefix"
-                             	},
-                              	"submodules/quickstart-microsoft-rdgateway"
-                           ]
-                        ]
+                        "Fn::Sub": "${QSS3KeyPrefix}submodules/quickstart-microsoft-rdgateway"
                     },
                     "RDGW1InstanceType": {
                         "Ref": "RDGW1InstanceType"
```

Comment:
RDGW1InstanceType is now RDGWInstanceType.
RDGW2InstanceType, RDGW1 and RDGW2 PrivateIPs and NetBIOS names are no longer parameters.

Please check out the latest in the RDGW. We made those changes there after popular demand to decrease from 2 instances to 1 by default but allow a way to deploy more.